src/index-notifications.html

```diff
@@ -0,0 +1,154 @@
+<!-- add something that does notifications + unread count -->
+<!doctype html>
+<html>
+  <head>
+    <title>ECP Demo</title>
+    <style>
+      html, body {
+        height: 100vh;
+        margin: 0;
+      }
+      .app {
+        position: relative;
+        height: 100%;
+      }
+      .description {
+        padding: 1rem;
+      }
+      .form-group {
+        padding: .4rem;
+      }
+      .ecp-frame {
+        position: fixed;
+        bottom: 1px;
+        right: 50px;
+        height: 40%;
+        width: 30%;
+        transition: all .5s ease;
+      }
+      .hidden {
+        height: 0;
+      }
+      .ecp-frame iframe {
+        border: 1px solid black;
+        border-bottom: none;
+        width: 100%;
+        height: 100%;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="app">
+      <div class="description">
+        <h1>Knowing how to handle ECP notifications</h1>
+        <p>
+          On ECP, it is possible to listen to some notifications through the listen method exposed by the SDK.
+          Thanks to this, you will be able to use all the notifications, including unread count
+        </p>
+        <p>The first thing you need to do is integrate ECP using the sdk:</p>
+        <xmp>
```

Comment:
is there a reason why we use the depreacted xmp tag instead of `<code>` or `<pre>`?

Reply:
That's because of me I believe 😅 . In other examples I use xmp so I didn't have to replace `<` and `>` with `&lt;` and `&gt;`.
But it's true that maybe we shouldn't have deprecated tags in examples...